console/network/src/canary_v0.rs

```diff
@@ -165,9 +160,17 @@ impl Network for CanaryV0 {
     /// The function name for the inclusion circuit.
     const INCLUSION_FUNCTION_NAME: &'static str = MainnetV0::INCLUSION_FUNCTION_NAME;
     /// The maximum number of certificates in a batch.
+    #[cfg(any(test, feature = "test"))] // TODO: or should we use the 'test-helpers' feature?
```

Comment:
perhaps we should consider joining both features?

Reply:
One day, one day. Removed the comments, I just wanted to see if CI would pass as is: 70c4ec66a